tools/sizing/size_finder.go

```diff
@@ -0,0 +1,103 @@
+package sizing
+
+import (
+	"fmt"
+
+	"github.com/Layr-Labs/eigenda/encoding"
+)
+
+// next_power_of_2(len(payload)*32/31)
+
+// BlobHeaderSize is the size of the header data on a blob, in bytes.
+const BlobHeaderSize = uint64(31)
+
+// PayloadSizeToBlobSize takes a payload size in bytes and returns the corresponding blob size in bytes.
+// The blob size is the size used for determining payments and throttling by EigenDA. Two payloads of
+// differing length that have the same blob size cost the same and use the same amount of bandwidth.
+func PayloadSizeToBlobSize(payloadSize uint64) uint64 {
```

Comment:
unused

Reply:
Now used as an extra sanity check in unit tests.